org.gridtracker.GridTracker.yml

```diff
@@ -0,0 +1,49 @@
+app-id: org.gridtracker.GridTracker
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: GridTracker
+rename-desktop-file: GridTracker.desktop
+rename-icon: gridtracker
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
+cleanup:
+  - /man
+  - /include
+  - /share/man
+  - /share/doc
+  - /doc
+  - '*.a'
+  - '*.la'
+  - /bin/usr
+modules:
+  - name: prepare
+    buildsystem: simple
+    build-commands:
+       - mkdir -p $FLATPAK_DEST/bin
+       - mkdir -p $FLATPAK_DEST/share/applications/
+       - mkdir -p $FLATPAK_DEST/share/icons/hicolor/128x128/apps/
+  - name: gridtraker
+    buildsystem: simple
+    build-commands:
+       - cp -r * $FLATPAK_DEST/bin
```

Comment:
If GridTracker license is GPL, why don't you compile it from source code? Using precompiled binaries is not ideal if there is a source code that can be compiled without any issue. Also, compiling it would allow ARM builds etc.

Reply:
I don't know how it will be compiled
if you create a docker or how is this code

https://gitlab.com/gridtracker.org/gridtracker

Reply:
`buildsystem: simple` should work fine.

Reply:
`buildsystem: autotools` should work fine.

Reply:
So, according to [Arch Linux AUR page](https://aur.archlinux.org/packages/gridtracker-bin/), GridTracker is closed-source and no source code is available. The correct project license in AppStream metadata should be: `LicenseRef-proprietary`